### ISSUE TITLE
Issue with cisco.dnac.sda_fabric_border_device 

### ISSUE THREAD

**Issue by FlGirard** [cisco-en-programmability/dnacenter-ansible#49]
Hello,

I'm trying to use the module cisco.dnac.sda_fabric_border_device to add the border role to a device in a fabric, but I'm meeting different problems. 

First, there is a problem in the documentation of the module (https://cisco-en-programmability.github.io/dnacenter-ansible/main/plugins/sda_fabric_border_device_module.html#ansible-collections-cisco-dnac-sda-fabric-border-device-module), the syntax indicated for the creation of a request will not succeed. Some arguments are not where they should be (ex: virtualNetworkName).

Secondly, the "externalConnectivitySettings" argument should be sent as a list to the DNAC API but it seems only possible to pass dictionaries as arguments.

**Comment by fmunozmiranda** [comment]
Hey @FlGirard, Could you provide us with the following information to have a clearer context of the problem:
**Environment:**
* Cisco DNA Center Version and patch:
* AnsibleVersion:
*Collection version:
*OSVersion:

**Screenshots**
Please provide a screenshot of the successful API call with cuRL, Postman, etc.

**Comment by FlGirard** [comment]
Sure

**Environment:**

- Cisco DNA Center Version and patch: 2.2.2.8
- Ansible Version: 5.1.0
- *Collection version: 5.1.2 
- *OSVersion: MacOs 12.2.1

**Screenshots:**

I need to make some changes to my network to make the request successful, but I find this screenshot relevant. In my DNA Center, I am able to send an API call and get a return with the reason for the failure (here I did not create the transit).


![image](https://user-images.githubusercontent.com/77703513/157917137-2ca4ce5a-8c58-4506-bf82-95df4a4034f7.png)

With the module, my call is malformed and gives me no description in return (I used the same parameters).

![image](https://user-images.githubusercontent.com/77703513/157917236-9a8e0367-7294-4015-a252-8470ca9e0668.png)

**Comment by zapodeanu** [comment]
You need an IP Transit to be configured to configure the border node. We do not have the API to configure that. It may be configured using the UI

**Comment by FlGirard** [comment]
Yes, I noticed that but that is not the point. I tried to make the same request: in DNAC and with the module.

The request sent by the module failed because of the malformed json (externalConnectivitySettings and virtualNetwork should be formatted as an array). Moreover, the documentation provides a malformed example. As the screenshot below shows:

![image](https://user-images.githubusercontent.com/77703513/158149760-08e19d94-85b7-4d30-90ba-19be4f644e20.png)

Also, I know it's probably not the good place for asking but why the API/Module and the GUI don't use the same requirements ? For example, to create a Border Node much more things are required with the API (a transit is not mandatory with the GUI).
